src/main/java/de/retest/recheck/ui/review/ScreenshotChanges.java

```diff
@@ -0,0 +1,34 @@
+package de.retest.recheck.ui.review;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import de.retest.recheck.report.ActionReplayResult;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.RootElementDifference;
+import de.retest.recheck.ui.image.Screenshot;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor( access = AccessLevel.PRIVATE )
+public class ScreenshotChanges {
+
+	private final Map<IdentifyingAttributes, Screenshot> screenshotMap;
+
+	public static ScreenshotChanges empty() {
+		return new ScreenshotChanges( Collections.emptyMap() );
```

Comment:
It's a best practise to use a constant variable instead of a new object. Saves memory if it's called often.